pylace/src/lib.rs

```diff
@@ -1027,19 +1029,20 @@ impl CoreEngine {
             checkpoint=None,
             transitions=None,
             save_path=None,
-            quiet=false,
+            update_handler=None,
         )
     )]
     fn update(
         &mut self,
+        py: Python<'_>,
```

Comment:
Is there a need for the `py` object if there is no `update_handler`? Is it still worth it to `allow_threads`?

Reply:
Yeah, during the update step, it allows other threads to run as it yields the GIL. This doesn't change things for us but it does make the code more usable to other users.